proc/cut_test.go

```diff
@@ -58,6 +65,21 @@ func TestCut(t *testing.T) {
 	proc.TestOneProc(t, fooAndBar, fooAndBar, "cut foo,bar")
 }
 
+func TestCutComplement(t *testing.T) {
```

Comment:
how about a test of `cut -c` on a record?

Reply:
added to TestCutNested